pom.xml

```diff
@@ -105,6 +105,7 @@
       <groupId>io.netty</groupId>
       <artifactId>netty-codec-haproxy</artifactId>
       <version>4.1.48.Final</version>
+      <optional>true</optional>
```

Comment:
we need an integration test without the `netty-codec-haproxy` jar, have a look at failsafe tests where a jar can be removed from the classpath to run the test in pom.xml